templates/zerver/development/integrations_dev_panel.html

```diff
@@ -95,8 +95,8 @@
             </div>
             <textarea id="fixture_body"></textarea>
             <div class="buttons">
-                <button id="send_all_fixtures_button" class="btn">Send all</button>
-                <button id="send_fixture_button" class="btn">Send!</button>
+                <button id="send_all_fixtures_button" class="bootstrap-btn">Send all</button>
+                <button id="send_fixture_button" class="bootstrap-btn">Send!</button>
```

Comment:
This seems to be the only use of `bootstrap-btn` in the portico pages, so potentially we can just add some styles for those buttons on this one development panel (if needed) and then move `bootstrap-btn.css` from `web/src/bundles/common.ts` to just the app bundle.

Reply:
Opened #32415 for the same.